dashboard/app/models/pd/application/teacher1819_application.rb

```diff
@@ -430,5 +430,57 @@ def meets_criteria
     def total_score
       response_scores_hash.values.map {|x| x.try(:to_i)}.compact.reduce(:+)
     end
+
+    # Called once after the application is submitted, and the principal approval is done
+    def auto_score
```

Comment:
Please add a comment about what it does in addition to when it should be called, and also mention that it's idempotent and won't override existing scores.

Reply:
Since this mutates the application object, should we call it `auto_score!`